app/models/ability.rb

```diff
@@ -42,10 +42,11 @@ def initialize(user)
     if user.role? :tutor
```

Comment:
> it was always coach..suddenly: tutor..:x

We originally named it `tutor` because that's how it was named in one of our user stories. But `coach` it is I guess?